src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/NodeMonitoringFailoverPlugin.java

```diff
@@ -95,23 +96,33 @@ void init(
       Log log,
       IMonitorServiceInitializer monitorServiceInitializer) {
     if (connection == null) {
-      throw new NullArgumentException("connection");
+      throw new NullArgumentException(Messages.getString(
```

Comment:
Could be wrong, but it looks like you could use a function that takes  a String[] and would generate a Messages.getString() object, so your code becomes lighter in the throws

Reply:
Oh, my bad. There isn't one. I was suggesting for you create one, so all your throw calls are oneliners.

Reply:
I can create one within `NodeMonitoringFailoverPlugin.java` and directly return a NullArgumentException object. I think adding a method within the `Messages` class wouldn't be as beneficial.